scripts/gods/god.gd

```diff
@@ -0,0 +1,28 @@
+class_name God
+extends Node2D
+
+signal favor_has_changed(favor: int)
+signal favor_cap_reached
+
+@export var god_data: RGod
+var favor: int = 0
```

Comment:
Favor should not exist on the god, should access via the material warehouse.

Reply:
It can live on the warehouse or on the god, I don't think it matter much to me, but if it makes more sense to you for it to be in the warehouse, I can move it.

Reply:
This hasnt been updated but going to pass it through and I can refactor as a part of my changes.